otm/resources/schemas/otm_schema.json

```diff
@@ -49,18 +49,18 @@
             "properties": {
                 "name": {"type": "string"},
                 "id": {"type": "string"},
-                "description": {"type": "string"},
+                "description": {"type": ["string", "null"]},
                 "risk": {
-                    "type": "object",
+                    "type": ["object", "null"],
                     "required": ["confidentiality", "integrity", "availability"],
                     "properties": {
                         "confidentiality": {"type": "number"},
                         "integrity": {"type": "number"},
                         "availability": {"type": "number"},
-                        "comment": {"type": "string"}
+                        "comment": {"type": ["string", "null"]}
                     }
                 },
-                "attributes": {"type": "object"}
+                "attributes": {"type": ["object", "null"]}
             }
         },
         "trustZones": {
```

Comment:
"trustZones" is an optional field so a null value should be allowed. Its "parent" field should also be optional (is this intended?)